core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java

```diff
@@ -893,7 +889,8 @@ public Response createObjectOrUploadPart(@HeaderParam("Content-MD5") final Strin
               .setMode(PMode.newBuilder()
                   .setOwnerBits(Bits.ALL)
                   .setGroupBits(Bits.ALL)
-                  .setOtherBits(Bits.NONE).build());
+                  .setOtherBits(Bits.NONE).build())
+              .setOverwrite(true);
```

Comment:
alignment

Reply:
it need to alignment with line 889, should not align with line 892